File: codeforces/469A - I wanna be the Guy/a.cpp
#include <bits/stdc++.h>
using namespace std;    
int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int n;
    cin >> n;
    vector<char> seen(n+1,0);
    int p; cin >> p;
    for(int i = 0; i < p; ++i) {
        int x; cin >> x;
        seen[x] = 1;
    }
    int q; cin >> q;
    for(int i = 0; i < q; ++i) {
        int x; cin >> x;
        seen[x] = 1;
    }
    bool ok = true; 
    for(int i = 1; i <= n; ++i) {
        if(!seen[i]) {
            ok = false;
            break;
        }
    }
    cout << (ok ? "I become the guy." : "Oh, my keyboard!");
    return 0;

}

File: codeforces/469A - I wanna be the Guy/a.py
def main():
    n = int(input().strip())
    p = set(map(int, input().split()[1:]))
    q = set(map(int, input().split()[1:]))
    print("I become the guy." if len(p | q) == n else "Oh, my keyboard!")
if __name__ == "__main__":
    main()